supplier_management_site/auth/eventbrite_ar.py

```diff
@@ -0,0 +1,25 @@
+from django.views.i18n import get_formats
+from social_core.backends.eventbrite import EventbriteOAuth2
+from django.conf import settings
+from django.views import i18n
+from django.utils.translation import to_locale, get_language
+
+
+class EventbriteAuth2(EventbriteOAuth2):
```

Comment:
You should use a more meaningful class name that explains better what you are trying to achieve. Maybe something like LanguageAwareEventbriteOAuth2?

Reply:
OK, i changed the name of the class